flags-sdk/hypertune/README.md

```diff
@@ -0,0 +1,42 @@
+# Flags SDK Template
+
+This example uses a template adapter for feature flags with the [Flags SDK](https://flags-sdk.dev) that works with the [Flags Explorer](https://vercel.com/docs/workflow-collaboration/feature-flags/using-vercel-toolbar).
+
+## Demo
+
+[https://hypertune-flags-sdk.vercel.app/](https://hypertune-flags-sdk.vercel.app/)
```

Comment:
This should be `flags-sdk-hypertune` for consistency, as others are for example https://flags-sdk-statsig.vercel.app/